doc/Release_Build_Creation.rst

```diff
@@ -113,7 +113,7 @@ Procedure
 #. Follow this `Guide <Provision Release Build.md>`_ to run your build.
 
 Tested by:
-
+- May 19, 2021: Justin Woo (justin.woo@seagate.com) on  "CentOS 7.8.2003 x86_64" VM running on a Windows 10 Desktop running VMware Workstation 16 Pro.
```

Comment:
It has to be formatted correctly

Reply:
Thanks @mukul-seagate11 I formatted it like how the rest did.